.github/workflows/oaf-check.yml

```diff
@@ -0,0 +1,17 @@
+name: Check Open API Framework Version
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "master" ]
```

Comment:
afair we've decided to include all branches for the CI
But it could be moved to the separate PR

Reply:
Yeah, this is an old test version. The up-to-date version is in objects API